wayback/tests/test_client.py

```diff
@@ -2,6 +2,8 @@
 from pathlib import Path
 import pytest
 import vcr
+import requests
+import mock
```

Comment:
Since this project requires Python 3.6+, you can just use the built-in `unittest.mock` library rather than adding a new dependency. `mock` is a backport of the built-in for older versions of Python.

Reply:
Huh, I didn't know that mock came build in with unittest.
I've changed that (and the typos below).